content/applications/marketing/events/sell_tickets.rst

```diff
@@ -1,68 +1,137 @@
-:guilabel:`Online Ticketing` features.
+In order to sell event tickets in Odoo, some settings must first be enabled.
 
-The :guilabel:`Tickets` feature allows tickets to be sold for an event.
+First, navigate to :menuselection:`Events app --> Configuration --> Settings`. In the
+:guilabel:`Registration` section, there are two different settings: :guilabel:`Tickets` and
+:guilabel:`Online Ticketing`.
 
-The :guilabel:`Online Tickets` feature allows for the sale of tickets to occur through the website.
+The :guilabel:`Tickets` setting allows users to sell event tickets with standard sales orders.
+
+The :guilabel:`Online Ticketing` setting allows users to sell event tickets online through their
+integrated Odoo website.
+
+To activate one (or both) of these settings, tick the checkbox beside the desired features, and
+click :guilabel:`Save` to implement those settings.
 
 .. note::
-   If these options are *not* enabled, a default :guilabel:`Register` button will be available for
-   free registrations.
+   If these options are *not* enabled, a default :guilabel:`Register` button becomes available for
+   visitors to interact with and procure free registrations to the event.
 
 .. image:: sell_tickets/events-settings-tickets.png
    :align: center
    :alt: View of the settings page for Odoo Events.
 
-Sell tickets through sales orders
-=================================
+With those settings enabled, Odoo automatically creates a new *Product Type* called, *Event Ticket*,
+which is accessible on every product form. Odoo also creates three event registration products (with
+the *Product Type* set to *Event Ticket*) that can be used or modified for event tickets.
+
+.. important::
+   When creating a new event registration product, the *Product Type* **must** be set to *Event
+   Ticket* in order for it to be selected in the *Product* column in the *Tickets*  tab on an event
+   form.
+
+   .. image:: sell_tickets/events-tickets-registration-product.png
+      :align: center
+      :alt: View of an event form highlighting the column product under the tickets tab in Odoo.
+
+.. note::
+   Any event with tickets sold either with standard sales orders, or online, features a
+   :guilabel:`Sales` smart button at the top of the event form.
+
+   .. image:: sell_tickets/events-sales-smartbutton.png
+      :align: center
+      :alt: View of an event's form and the sales smart button in Odoo Events.
+
+   Clicking that smart button reveals a separate page, showcasing all the sales orders (standard
+   and/or online) related to tickets that have been sold for that specific event.
 
-In the :guilabel:`Sales` application, choose a previously-created Event Registration (as if it were
-a product), and add it as a product line. Upon adding the registration, a pop-up appears, which
-allows for the selection of a specific event (and ticket tier). That specific event ticket is then
-attached to the sales order.
+Sell event tickets with sales orders
```

Comment:
Same distinction in the last comment.
All events tickets involve sales orders, the difference is the apps used to generate them. 

Consider editing the language to this and the next section accordingly.

```suggestion
Sell event tickets with the Sales app
```